apps/console/src/features/organizations/components/add-organization-modal.tsx

```diff
@@ -68,15 +69,16 @@ export const AddOrganizationModal: FunctionComponent<AddOrganizationModalPropsIn
 
     const { t } = useTranslation();
 
-    const dispatch = useDispatch();
+    const dispatch: Dispatch<any> = useDispatch();
```

Comment:
```suggestion
    const dispatch: Dispatch = useDispatch();
```

Reply:
Check this btw :D

Reply:
Fixed by - https://github.com/wso2/identity-apps/pull/3792/commits/913b79b7ab6c4605441b5d57363c7e722c0c1b1e